AK/SourceGenerator.h

```diff
@@ -56,6 +57,11 @@ class SourceGenerator {
         m_mapping.set(key, move(value));
     }
 
+    void set(StringView key, FlyString const& value)
+    {
+        set(key, value.to_string());
+    }
+
```

Comment:
The SourceGenerator changes should probably go in their own commit, for the sake of being atomic - they're useful independently.